addons/crm/models/crm_lead.py

```diff
@@ -2085,14 +2066,33 @@ def _is_rule_based_assignment_activated(self):
     # MAILING
     # ------------------------------------------------------------
 
+    # A few options may allow to show the message:
+    # - Have a method on mail thread to show creation message (on conditions) to be inherited
+    # - Or Remove subtype here in those conditions
+    # - Or always show certain subtype' messages?
+    # - or on the subtype directly (but would say no to that!)
+    # can we just remove it ? _message_parse_extract_payload_postprocess: should not be an issue since not coming from mail ?
+    # But we would probably want still the same creation subtype (thinking about subscriptions, yet hidden subtype)
+
+    # Question: could we ever have a different subtype? (-> is content of _creation_message never displayed?)
     def _creation_subtype(self):
         return self.env.ref('crm.mt_lead_create')
 
     def _creation_message(self):
         self.ensure_one()
+        # Quid of portal, is this accessed on front end? (and hidden?)
+        team_msg = _('A new lead has been created and is not assigned to any team.')
         if self.team_id:
-            return _('A new lead has been created for the team "%(team_name)s".', team_name=self.team_id.display_name)
-        return _('A new lead has been created and is not assigned to any team.')
+            team_msg = _('A new lead has been created for the team "%(team_name)s".', team_name=self.team_id.display_name)
+        if self.partner_id:
+            return team_msg
+        if intial_values_msg := self._initial_values_creation_message():
+            return Markup("<br>").join([team_msg, intial_values_msg])
+        return team_msg
+
+    def _display_creation_message_with_subtype(self):
```

Comment:
Would avoid, as mail thread code is already complex. Better remove invisible stuff (summary hack), display, maybe hide for portal, and see how it goes, then solves potential issues or consider it as a technical limitation. Too much overrides make code hard to understand :) .